roles/docker/templates/docker-options.conf.j2

```diff
@@ -1,3 +1,4 @@
 [Service]
 Environment="DOCKER_OPTS={{ docker_options | default('') }} \
 --iptables=false"
+MountFlags=shared
```

Comment:
I don't think this can be a default due to limitations of only having shared mounts in some environments, but I'm fine if this becomes an opt-in.